src/Query/Filter/HeadingField.ts

```diff
@@ -5,13 +5,6 @@ import { TextField } from './TextField';
  *
  */
 export class HeadingField extends TextField {
```

Comment:
There's no change in the HeadingField test.
Please add a single, simple case that confirms it is hooked up for regex searching.

Reply:
I'll be adding tests of Heading with regex as part of documenting and testing this feature in #1015. So I'm resolving this comment.